modules/analysis-services/server/README.md

```diff
@@ -19,63 +19,30 @@ This module deploys an Analysis Services Server.
-| :-- | :-- | :-- |
-| `name` | string | The name of the Azure Analysis Services server to create. |
-
-**Optional parameters**
-
-| Parameter Name | Type | Default Value | Allowed Values | Description |
-| :-- | :-- | :-- | :-- | :-- |
-| `diagnosticEventHubAuthorizationRuleId` | string | `''` |  | Resource ID of the diagnostic event hub authorization rule for the Event Hubs namespace in which the event hub should be created or streamed to. |
-| `diagnosticEventHubName` | string | `''` |  | Name of the diagnostic event hub within the namespace to which logs are streamed. Without this, an event hub is created for each log category. |
-| `diagnosticLogCategoriesToEnable` | array | `[allLogs]` | `['', allLogs, Engine, Service]` | The name of logs that will be streamed. "allLogs" includes all possible logs for the resource. Set to '' to disable log collection. |
-| `diagnosticMetricsToEnable` | array | `[AllMetrics]` | `[AllMetrics]` | The name of metrics that will be streamed. |
-| `diagnosticSettingsName` | string | `''` |  | The name of the diagnostic setting, if deployed. If left empty, it defaults to "<resourceName>-diagnosticSettings". |
-| `diagnosticStorageAccountId` | string | `''` |  | Resource ID of the diagnostic storage account. |
-| `diagnosticWorkspaceId` | string | `''` |  | Resource ID of the diagnostic log analytics workspace. |
-| `enableDefaultTelemetry` | bool | `True` |  | Enable telemetry via a Globally Unique Identifier (GUID). |
-| `firewallSettings` | object | `{object}` |  | The inbound firewall rules to define on the server. If not specified, firewall is disabled. |
-| `location` | string | `[resourceGroup().location]` |  | Location for all Resources. |
-| `lock` | string | `''` | `['', CanNotDelete, ReadOnly]` | Specify the type of lock. |
-| `roleAssignments` | array | `[]` |  | Array of role assignment objects that contain the 'roleDefinitionIdOrName' and 'principalId' to define RBAC role assignments on this resource. In the roleDefinitionIdOrName attribute, you can provide either the display name of the role definition, or its fully qualified ID in the following format: '/providers/Microsoft.Authorization/roleDefinitions/c2f4ef07-c644-48eb-af81-4b1b4947fb11'. |
-| `skuCapacity` | int | `1` |  | The total number of query replica scale-out instances. |
-| `skuName` | string | `'S0'` |  | The SKU name of the Azure Analysis Services server to create. |
-| `tags` | object | `{object}` |  | Tags of the resource. |
-
-
-## Outputs
+## Usage examples
 
-| Output Name | Type | Description |
-| :-- | :-- | :-- |
-| `location` | string | The location the resource was deployed into. |
-| `name` | string | The name of the analysis service. |
-| `resourceGroupName` | string | The resource group the analysis service was deployed into. |
-| `resourceId` | string | The resource ID of the analysis service. |
+The following module usage examples are retrieved from the content of the files hosted in the module's `tests` folder.
```

Comment:
Could this be reworded for example to not necessarily call out the test folder? What if a user has access to a readme, and the BR/TS, but not the source code? 

E.g. '_The following section provides usage examples for the module, by inputting various parameter combinations, which were then used to validate and deploy the module successfully.'_

Thoughts?

Reply:
@eriqua, I think Ahmad is on to something

Reply:
I wonder though if we shouldn't still point them somewhere as users would otherwise have no idea. We could for example link the "to be" public registry page

Reply:
Good points both! I agree we should somehow ref the module source code. The challenge is the version, which is not tracked by the readme, so at this stage we cannot be very specific, but just point to the source code repo in general. I'm fine with @AlexanderSehr suggestion. And thanks @ahmadabdalla for raising this!

Reply:
Updated in all PRs

Reply:
Yep all good @AlexanderSehr and @eriqua ✌️